main.py

```diff
@@ -56,6 +58,7 @@ def main(WorldClass):
             clock.tick(FRAME_RATE)
             display.flip()
             pygame.display.set_caption(TITLE + ' ' + world.get_info())
+        world.end()
```

Comment:
Je roept hier world.end aan maar ik zie end nergens gedefinieerd worden.

Reply:
Mmm, in de code wel zie ik. Maar niet in de file changes hier. 
Hadden we .end al kennelijk?

Reply:
Volg ik niet helemaal - hier beneden zie je m toch in t groen gehighlight? Of in t files changed tabje?

Reply:
Nu zie ik hem wel.